Automattic/wp-calypso
# Reader: add PostComment to devdocs w/ new classNames

This PR adds the `PostComment` class to our devdocs so that we can parallelize styling efforts with the rest of implementation.  It also adds in a new prop `displayType` with three possible values:

|key |css class | meaning|
|---- |---|---|
|full | `is-full` | default behavior. show the full comment|
|excerpt | `is-excerpt` | default behavior for convo tool.  show 3 lines by default w/ an option for read more |
| singleLine | `is-single-line` | only show a single line of the text. [see here](https://github.com/Automattic/wp-calypso/issues/16933) |

The devdocs postcomments look like this:

```
singleLine
 - excerpt
full
```

To Test: 
1. go to: http://calypso.localhost:3000/devdocs/blocks/post-comment and verify you see PostComments there.  as well as is-single-line, is-full, and is-excerpt comments
2. verify that fullpost still works as expected.
